pkg/topology/topology.go

```diff
@@ -12,22 +12,45 @@ import (
 	"github.com/ethersphere/bee/pkg/swarm"
 )
 
-var ErrNotFound = errors.New("no peer found")
-var ErrWantSelf = errors.New("node wants self")
+var (
+	ErrNotFound         = errors.New("no peer found")
+	ErrAlreadyConnected = errors.New("peer already connected")
```

Comment:
i'm not so sure we need this. due to the structure of the `manage` loop, it could even be that we can remove the `error` from the `AddPeer` function signature. i am not sure it is necessary that much